app/views/deploys/show.html.erb

```diff
@@ -33,6 +33,7 @@
 <section class="clearfix tabs">
   <div class="tab-content deploy-details">
     <div class="tab-pane active" id="output" data-stream-url="<%= stream_path(@deploy.job) %>" data-desktop-notify="<%= current_user.desktop_notify? %>">
+      <%= Samson::Hooks.render_views(:deploy_view, self, deploy: @deploy, project: @project) %>
```

Comment:
since they are @ variables we don't need to pass them ... might still be nice to do just as an api ...